content/setup/_index.md

```diff
@@ -12,7 +12,7 @@ To set up your own File-picker instance, you need to do the following:
 - Edit the configuration files
 - Set up a web server
 
-This guide has been tested in CentOS 7 and Ubuntu 22.04, but it should work in
+This guide has been tested in CentOS 7, Ubuntu 22.04 and Linux Mint 21.1, but it should work in
```

Comment:
Linux Mint is based on Ubuntu, I don't think they are different enough to deserve to be distinguished here